### PR TITLE
Improved home subscriptions

### PR DESCRIPTION
**Pull request by nargacu83**
Hi!

I've changed the behavior of the home `Subscriptions` component to behave more like the `VideosContainer` component. This improves #2735 and was asked by @ewoks in https://github.com/ViewTube/viewtube/issues/2587#issuecomment-2081570870.

It now shows a `Show more` button with a similar behavior like the trending videos section. Subscriptions videos will Auto expand when trending videos are hidden and hides the `Show more` button.

### CODE REVIEW

**Comment by moisout** [comment]
Very cool, Thanks! I'll take a look it ASAP.

File: client/components/home/Subscriptions.vue [hunk 1]
```diff
@@ -1,22 +1,53 @@
...
 
+const settingsStore = useSettingsStore();
 const { data: subscriptions, pending: subscriptionsLoading } = useGetUserSubscriptions({
-  limit: 4
+  limit: 20
+});
+
+const showMore = ref(false);
+const displayedSubscriptions = computed(() => {
+  if (subscriptions && subscriptions.value.videos) {
```
Comment (moisout) [inline client/components/home/Subscriptions.vue]:
```suggestion
  if (subscriptions.value && subscriptions.value.videos) {
```
Please add .value here ;)

Comment (moisout) [reply client/components/home/Subscriptions.vue]: Otherwise it looks great!

Comment (nargacu83) [reply client/components/home/Subscriptions.vue]: Done!

**Comment by moisout** [comment]
Will merge

**Comment by ewoks** [comment]
guess this comes in next release (viewtube.io or container or both? ), I will check and report back, so we close ;) 
Thanks a lot

**Pull request merged**